src/Form/Element/Factory/CurrencySelectFactory.php

```diff
@@ -2,7 +2,7 @@
 
 namespace ZFBrasil\DoctrineMoneyModule\Form\Element\Factory;
 
-use Zend\Form\FormElementManager;
+use Interop\Container\ContainerInterface;
```

Comment:
Please use the `ContainerInterface` from PSR-11 instead https://github.com/php-fig/container/blob/master/src/ContainerInterface.php.